app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -236,6 +238,7 @@ public void onCreate() {
             Logger.registerLogger(new AndroidLogger(this.getGlobalStorage(AndroidLogEntry.STORAGE_KEY, AndroidLogEntry.class)));
             pil.dumpToNewLogger();
         }
+        XPathErrorLogger.registerStorage(getGlobalStorage(XPathErrorEntry.STORAGE_KEY, XPathErrorEntry.class));

```

Comment:
@phillipm Related to the conversation we were having yesterday and maybe we just need to sit down and go through this, but probably want this to be app storage instead of global storage?